api/v1alpha1/stage_types.go

```diff
@@ -301,6 +301,13 @@ type FreightSources struct {
 	// made available to the Stage. This field is optional. When left unspecified,
 	// the field is implicitly treated as if its value were "OneOf".
 	//
+	// Accepted Values:
+	//
+	// - "All": Freight must be be verified and, if applicable, soaked in all
+	//   upstream Stages to be considered available for promotion.
+	// - "OneOf": Freight must be verified and, if applicable, soaked in at least
+	//    one upstream Stage to be considered available for promotion.
+	// - "": Treated the same as "OneOf".
```

Comment:
Sorry. I know this means running codegen again, but these are all more readable if we leave a line between the free-form comments and the kubebuilder markers.
```suggestion
	// - "": Treated the same as "OneOf".

```